keyboards/kprepublic/bm40hsrgb/keymaps/wolff_abnt2/config.h

```diff
@@ -0,0 +1,123 @@
+/* Copyright 2020 tominabox1
```

Comment:
This isn't you, right? So the license header should get a change

Reply:
Fixed.